nilearn/conftest.py

```diff
@@ -1,20 +1,12 @@
 """Configuration and extra fixtures for pytest."""
 
-import nibabel
 import numpy as np
 import pandas as pd
 import pytest
 from nibabel import Nifti1Image
 from scipy.signal import get_window
 
-from nilearn import image
 from nilearn._utils.helpers import is_matplotlib_installed
-
-# we need to import these fixtures even if not used in this module
-from nilearn.datasets.tests._testing import (
-    request_mocker,  # noqa: F401
-    temp_nilearn_data_dir,  # noqa: F401
-)
```

Comment:
this would use those fixture for ALL our tests and they too use monkeypatch making free threading testing impossible